backend/app/routes/admin/index.js

```diff
@@ -1,14 +1,19 @@
 const router = require('express').Router({ mergeParams: true });
 const validationMiddleware = require('../../../helpers/middlewares/validation');
-const postSuperAdminSchema = require('./@validationSchema');
+const {postSuperAdminSchema ,putSuperAdminSchema }= require('./@validationSchema');
 const createSuperAdmin = require('./createSuperAdmin');
 const postSuperAdmin = require('./postSuperAdmin');
+const putSuperAdmin = require('./update_data');
 
 router.get('/createSuperAdmin', createSuperAdmin);
 router.post(
   '/superAdmin',
   validationMiddleware(postSuperAdminSchema),
   postSuperAdmin
 );
-
+router.put(
+  '/:id',
+  validationMiddleware(putSuperAdminSchema),
+  putSuperAdmin
```

Comment:
by default, you are using all the admin's profile as a super admin(boolean field you have taken is true for all). So either of the updated profile will work, right?
I must req you to think and check your backend again you guys have built.

Reply:
Yes, by the comment what I meant was tor rename it to `putAdminSchema`.
No there is no default assumption like that. For `superAdmin` the boolean is true else it will be false for admins.